Aulas-HTML/exercicios/aula2/index.html

```diff
@@ -21,3 +21,119 @@
  - Abrir um pull request para o repositório original (da Womakers)
  - Acompanhar o prazo e marcar a entrega no Classroom
 -->
+!DOCTYPE html>
+<html>
+
+<head>
+  <html lang="pt-BR">
+
+  </html>
```

Comment:
Oi, Audry 😊 
Toma cuidado pra não esquecer de fechar as tags ou fechar fora da hora, tudo bem? Percebi que você fechou a tag html logo aqui no início!